rskj-core/src/integrationTest/java/co/rsk/snapshotsync/SnapshotSyncIntegrationTest.java

```diff
@@ -0,0 +1,113 @@
+package co.rsk.snapshotsync;
+
+import co.rsk.util.cli.NodeIntegrationTestCommandLine;
+import co.rsk.util.cli.ConnectBlocksCommandLine;
+import co.rsk.util.FilesHelper;
+import co.rsk.util.cli.RskjCommandLineBase;
+import co.rsk.util.ThreadTimerHelper;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.apache.commons.lang3.RandomUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.junit.jupiter.api.Test;
+
+import java.io.*;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static co.rsk.util.FilesHelper.readBytesFromFile;
+import static co.rsk.util.FilesHelper.writeBytesToFile;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class SnapshotSyncIntegrationTest {
+    private static final String TAG_TO_REPLACE_SERVER_RPC_PORT = "<SERVER_NODE_RPC_PORT>";
+    private static final String TAG_TO_REPLACE_SERVER_PORT = "<SERVER_NODE_PORT>";
+    private static final String TAG_TO_REPLACE_SERVER_DATABASE_PATH = "<SERVER_NODE_DATABASE_PATH>";
+    private static final String TAG_TO_REPLACE_NODE_ID = "<SERVER_NODE_ID>";
+    private static final String TAG_TO_REPLACE_CLIENT_DATABASE_PATH = "<CLIENT_NODE_DATABASE_PATH>";
+    private static final String TAG_TO_REPLACE_CLIENT_PORT = "<CLIENT_PORT>";
+    public static final String TMP_RSKJ_INTEGRATION_TEST_FOLDER = "rskj-integration-test-";
+    private final String rskConfFileServerName = "snap-sync-server-rskj.conf";
+    private final String rskConfFileClientName = "snap-sync-client-rskj.conf";
+    private final int portServer = RandomUtils.nextInt(1, 5000);
+    private final int portClient = RandomUtils.nextInt(5001, 9999);
+    private final String pathDatabaseServer = "/.rsk/regtest/database";
+    private final String randomPathDatabaseClient = RandomStringUtils.randomAlphanumeric(10);
+
+    @Test
+    public void whenStartTheServerAndClientNodes_thenTheClientWillSynchWithServer() throws IOException, InterruptedException {
+        //given
+        Path tempDirectory = Files.createTempDirectory(TMP_RSKJ_INTEGRATION_TEST_FOLDER);
+        Path dirDatabaseServer = Paths.get(System.getProperty("user.home") + pathDatabaseServer);
```

Comment:
Yes, there is a reason, the ConnectBlocks tool just connect the data on on the home folder. You can't choose the path that it will use, it will always be the default one for mainnet, testnet and regtest.